src/LoginView.js

```diff
@@ -129,7 +129,7 @@ export default class LoginView extends PureComponent {
 
     return (
       <WebView
-        userAgent="Mozilla/5.0 (Macintosh; Intel Mac OS X 10.13; rv:61.0) Gecko/20100101 Firefox/61.0"
+        userAgent="Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_6) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/13.1.2 Safari/605.1.15"
```

Comment:
Please verify if unit tests still pass, specially the snapshot

Reply:
updated snapshot, confirmed all tests pass